cds_ils/vocabularies/data/tags.json

```diff
@@ -13,5 +13,30 @@
     "type": "tag",
     "key": "LEGAL_SERVICE_LIBRARY",
     "text": "Legal service Library"
+  },
+  {
+    "type": "tag",
+    "key": "ENGLISH_BOOK_CLUB",
+    "text": "English book club"
+  },
+  {
+    "type": "tag",
+    "key": "ARCHIVES",
+    "text": "Archives"
+  },
+  {
+    "type": "tag",
+    "key": "YELLOW_REPORT",
```

Comment:
yellow report will not be a tag, as we discussed it will be a series and included in the second part of this task